airflow/configuration.py

```diff
@@ -186,6 +186,18 @@ class AirflowConfigParser(ConfigParser):
         ('logging', 'worker_log_server_port'): ('celery', 'worker_log_server_port', '2.2.0'),
         ('api', 'access_control_allow_origins'): ('api', 'access_control_allow_origin', '2.2.0'),
         ('api', 'auth_backends'): ('api', 'auth_backend', '2.3'),
+        ('database', 'sql_alchemy_conn'): ('core', 'sql_alchemy_conn', '2.3'),
+        ('database', 'sql_engine_encoding'): ('core', 'sql_engine_encoding', '2.3'),
+        ('database', 'sql_engine_collation_for_ids'): ('core', 'sql_engine_collation_for_ids', '2.3'),
+        ('database', 'sql_alchemy_pool_enabled'): ('core', 'sql_alchemy_pool_enabled', '2.3'),
+        ('database', 'sql_alchemy_pool_size'): ('core', 'sql_alchemy_pool_size', '2.3'),
+        ('database', 'sql_alchemy_max_overflow'): ('core', 'sql_alchemy_max_overflow', '2.3'),
+        ('database', 'sql_alchemy_pool_recycle'): ('core', 'sql_alchemy_pool_recycle', '2.3'),
+        ('database', 'sql_alchemy_pool_pre_ping'): ('core', 'sql_alchemy_pool_pre_ping', '2.3'),
+        ('database', 'sql_alchemy_schema'): ('core', 'sql_alchemy_schema', '2.3'),
+        ('database', 'sql_alchemy_connect_args'): ('core', 'sql_alchemy_connect_args', '2.3'),
+        ('database', 'load_default_connections'): ('core', 'load_default_connections', '2.3'),
+        ('database', 'max_db_retries'): ('core', 'max_db_retries', '2.3'),
     }
 
     # A mapping of old default values that we want to change and warn the user
```

Comment:
Still - we miss the deprecations. All the conf values that have been deprecated should be added to this list

Reply:
We have a depreciation warnings. See:  https://github.com/apache/airflow/pull/22284/files#r834137403

Reply:
🤦  indeed. Bad github folding